database/mongodb/mongodb.go

```diff
@@ -77,9 +78,6 @@ func (m *Mongo) Open(dsn string) (database.Driver, error) {
 		return nil, err
 	}
 	migrationsCollection := purl.Query().Get("x-migrations-collection")
-	if len(migrationsCollection) == 0 {
```

Comment:
default config value fallback removed

Reply:
Actually I did that on purpose, forgot to mention it I guess. 'Twas a long day.

The reason for this change is that the fallback was set both in `WithInstance` and in `Open`.
From the code, it would seem that it only needs to be set in `WithInstance` as no step in `Open`, before the `WithInstance` test, requires the field.
It could also be moved out of `WithInstance` and into `Open` but I believe it makes sense to have it in `WithInstance`.